www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+of substantially all of Gratipay's assets, Gratipay may assign its rights and
+obligations under these Terms to its successor.
+
+  1. Modifications to this Agreement: Gratipay may occasionally revise these
+     Terms. You can find the most recent version at
+http://gratipay.com/about/terms/. If we determine, in our sole discretion, that
+a change to these Terms is material, we will notify you of the change through
+the Service or by emailing you at the address associated with your account. By
+using the Service after a new version of these Terms becomes effective, you
+thereby agree to be bound by the revised Terms.
+
+  1. Choice of Law: These Terms and the relationship between you and Gratipay
+     shall be governed by the laws of the State of Pennsylvania without regard
+to its conflict of law provisions. Any and all claims, causes of action, or
+disputes (regardless of theory) arising out of or relating to the Service,
+shall be brought exclusively in the courts located in Allegheny County,
+Pennsylvania, or the U.S. District Court for the Western District of
+Pennsylvania. You consent and waive any objection to the jurisdiction of and
+venue in these courts.
+
+  1. Authority to Enter Into Agreement: You represent and warrant that you have
+     the legal authority to enter into this agreement and that the performance
+of obligations under this agreement will not violate any agreement between you
+and any other person, firm, or organization. If you are entering this agreement
+on behalf of another person or entity, you represent and warrant that you are
+authorized to do so.
+
+  1. Export Controls: You hereby represent and warrant that (i) you are not
+     located in a country that is subject to a U.S. Government embargo, or that
+has been designated by the U.S. Government as a "terrorist supporting" country;
+and (ii) you are not listed on any U.S. Government list of prohibited or
+restricted parties.
+
+  1. Third-Party Beneficiaries: There are no other third-party beneficiaries to
+     these Terms.
+
+  1. Waiver and Severability: Failure by Gratipay to enforce any right or
+     provision of these Terms will not be deemed a waiver of such right or
+provision. In the event that any provision of these Terms is held to be invalid
+or unenforceable, then that provision will be limited or eliminated to the
+minimum extent necessary, and the remaining provisions of these Terms will
+remain in full force and effect.
+
+  1. Entire Agreement: These Terms and our [Privacy
+     Policy](/about/policies/privacy) together form the entire and exclusive
+agreement between you and Gratipay regarding the Service (excluding any service
+for which you have a separate agreement with Gratipay that is explicitly in
+addition to or in place of these Terms), and these Terms supersede and replace
+any prior agreements between you and Gratipay regarding the Service. There
+shall be third party beneficiaries to the Terms.
```

Comment:
The very last sentence is confusing to me, in light of the third-to-last section, just above:

> Third-Party Beneficiaries: There are no other third-party beneficiaries to these Terms.

Is this last sentence right?